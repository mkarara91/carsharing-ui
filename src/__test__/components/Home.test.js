import * as React from "react";
import { shallow } from "enzyme";
import Home from "../../components/home";

describe("Testing Home Component", () => {
    it("Home component should match snapshot", () => {
        const comp = shallow(<Home />);
        expect(comp).toMatchSnapshot();
    });
});
