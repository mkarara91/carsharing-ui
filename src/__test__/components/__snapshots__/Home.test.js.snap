// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Home Component Home component should match snapshot 1`] = `ShallowWrapper {}`;
